StepsToCare/CareEntry.ascx.cs

```diff
@@ -130,6 +130,13 @@ namespace RockWeb.Plugins.rocks_kfs.StepsToCare
         DefaultBooleanValue = false,
         Key = AttributeKey.EnableCustomFollowUp )]
 
+    [CustomDropdownListField( "Follow Up Worker Assignment",
+        Description = "Enable the follow up worker to be assigned instead of just auto assigning the first worker added to need. Hide, existing behavior first worker gets assignment. Show, show a checkbox to assign follow up worker with first worker checked by default. Show and Assign All, show the checkbox and select all as default.",
```

Comment:
Test if HTML is supported in tooltips.
```suggestion
        Description = "Enable the follow up worker to be assigned instead of just auto assigning the first worker added to need. Hide: First worker gets assignment (existing behavior). Show: Show a checkbox to assign follow up worker with first worker checked by default. Show and Assign All: Show the checkbox and select all as default.",
```

Reply:
Test if HTML is supported in tooltips.